infrastructure/smart-contracts/test/get-winning-key-count/WinningKeyCountSimulations.mjs

```diff
@@ -0,0 +1,115 @@
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import {getWinningKeyCountLocal} from "../utils/getWinningKeyCountLocal.mjs";
+import fs from "fs";
+
+
+function RunWinningKeyCountSimulations(deployInfrastructure) {
+
+    return function () {
+    it("Confirm the reward rate for submissions.", async function () {
+        const { addr1 } = await loadFixture(deployInfrastructure);
+
+        const stakingBoostFactors = [1, 5, 10, 100, 200, 300, 700];
+        const keyAmountTests = [1, 5, 10, 100, 200, 300, 1000]; // Test cases for staked key amounts
```

Comment:
Let include more boostFactors and more keyAmounts to test with.
We should also have a reference to waht it will be with 100K keys

Reply:
What range of boost factors should be added?

Reply:
How about something like `1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987,` for both keyCount and boosts, just in addition to what we have, might catch something weird